src/todolist.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { getFullTodos, getUserTodos } from "./api.js";
+import Filters from "./Filters";
+
+class TodoList extends React.Component {
+  state = {
+    todos: null,
+    filter: "",
+    reversed: false,
+  };
+
+  async componentDidMount() {
+    const todos = (this.props.userId === "") ? await getFullTodos() : await getUserTodos(this.props.userId);
```

Comment:
this.props.userId ? await getFullTodos() : await getUserTodos(this.props.userId);